src/init/autoDetectProject.ts

```diff
@@ -16,12 +16,7 @@ export function detectProject(
   }
 
   return findFiles('**/package.json', '**/node_modules/**')
-    .then((results) => {
-      return results && results.length ? results as model.IVSCodeUri[] : [];
-    },
-    (rej) => {
-      return [rej];
-    });
+    .then((results) => results && results.length ? results as model.IVSCodeUri[] : [], (rej) => [rej]);
```

Comment:
Agreed. Although I'm used to adding the parenthesis in every lamda expression I'll follow the project's coding guidelines.